### PR TITLE
Rnb 72/default field cleanup

### PR DESCRIPTION
**Pull request by shadmans**
# What does this MR do?

- Remove use of LocalizationConsumer from all default component.
    - Note: FormDisplay seems to use prop `value` to display error or normal text depending on type (thus we need screen prop after type check since we can't assume the screen)
- Add min height to the text and password inputs
- Add consistent styles to input/password input field

# Developer Merge Checklist

## General
Always fill out this section
1. [x] The title of the MR contains the Jira task number - if there is no task, create one
1. [x] If this MR depends on a different MR, it is clearly indicated in the description. Please use the ## tag along with a link to the other MR.
1. [x] Latest develop is merged into this branch
1. [x] MR is assigned reviewer and the task is moved to the Pending MR column on Jira
1. [x] All strings and their translation are added to the locales

## Maintainability
Always fill out this section
1. [x] If new tools have been added, the readme has been updated with info on them
1. [x] All debug code removed (i.e. console.log)

## Testing
Always fill out this section
1. [x] MR has been tested by the developer on Android
1. [ ] MR has been tested by the developer on iOS
1. [x] Pages with new/updated UI have snapshot tests

## Documentation & Readability
Always fill out this section
1. [x] Functions have in-line comments as needed
1. [x] Corner cases and complex algorithms are explained
1. [x] Incomplete code is marked with TODO or FIXME
1. [x] If there are UI changes, screenshots of the new changes are attached to this merge request.

## Package Management
If packages have been added or updated fill out this section
1. [x] New packages are added to dependencies or devDependencies in package.json
1. [x] Android release still builds
1. [ ] iOS release still builds

## Native Code
If native code has been added fill out this section
1. [ ] Android release still builds
1. [ ] iOS release still builds

### CODE REVIEW

File: src/forms/FormPasswordField.tsx [hunk 1]
```diff
@@ -47,14 +48,17 @@ export const FormPasswordField = React.memo(
...
             return (
-                <PasswordInput
-                    value={value}
-                    onChange={changeHandler}
-                    onBlur={blurHandler}
-                    errorMessage={error}
-                    errorMessageScreen="formErrors"
-                    {...rest}
-                />
+                <LocalizationConsumer>
```
Comment (shopthing-connor) [inline src/forms/FormPasswordField.tsx]: If we're intending on removing localization, should we not remove it from all the components and have the user implement it as needed?

File: src/forms/Form.tsx [hunk 2]
```diff
@@ -152,7 +152,7 @@ export class Form extends React.PureComponent<IForm, State> {
                 this.setError(path, errors[0]);
             });
 
-            this.setError('formError', defaultFormError);
+            this.setError('formError', translate(`formErrors.${defaultFormError}`));
```
Comment (shadmans) [inline src/forms/Form.tsx]: This is the only case of translate remaining as we are setting it seperately to yup
